doc/man/8/mount.ceph.rst

```diff
@@ -19,31 +18,29 @@ Description
 **mount.ceph** is a helper for mounting the Ceph file system on a Linux host.
 It serves to resolve monitor hostname(s) into IP addresses and read
 authentication keys from disk; the Linux kernel client component does most of
-the real work. In fact, it is possible to mount a non-authenticated Ceph file
-system without mount.ceph by specifying monitor address(es) by IP::
-
-        mount -t ceph 1.2.3.4:/ /mnt/mycephfs
-
-The first argument is the device part of the mount command. It includes host's
-socket and path within CephFS that will be mounted at the mount point. The
-socket, obviously, takes the form ip_address[:port]. If the port is not
-specified, the Ceph default of 6789 is assumed. Multiple monitor addresses can
-be passed by separating them by commas. Only one monitor is needed to mount
-successfully; the client will learn about all monitors from any responsive
-monitor. However, it is a good idea to specify more than one in case the one
-happens to be down at the time of mount.
-
-If the host portion of the device is left blank, then **mount.ceph** will
-attempt to determine monitor addresses using local configuration files
-and/or DNS SRV records. In similar way, if authentication is enabled on Ceph
-cluster (which is done using CephX) and options ``secret`` and ``secretfile``
-are not specified in the command, the mount helper will spawn a child process
-that will use the standard Ceph library routines to find a keyring and fetch
-the secret from it.
+the real work. To mount a Ceph file system use::
+
+        mount.ceph name@fs_name=/ /mnt/mycephfs -o mon_host=1.2.3.4
+
+The first argument is the device part of the mount command. It includes the
+RADOS user for authentication, the file system name and a path within CephFS
+that will be mounted at the mount point. Cluster ID is optional. Monitor addresses
+are passed using ``mon_host`` mount option. Multiple monitor addresses can be
+passed by separating them with a space. Only one monitor is needed to mount
+successfully; the client will learn about all monitors from any responsive monitor.
+However, it is a good idea to specify more than one in case the one happens to be
+down at the time of mount. Monitor addresses takes the form ip_address[:port]. If
+the port is not specified, the Ceph default of 6789 is assumed.
+
+If monitor addresses are not specified, then **mount.ceph** will attempt to determine
+monitor addresses using local configuration files and/or DNS SRV records. In similar
+way, if authentication is enabled on Ceph cluster (which is done using CephX) and
+options ``secret`` and ``secretfile`` are not specified in the command, the mount
+helper will spawn a child process that will use the standard Ceph library routines
+to find a keyring and fetch the secret from it.
```

Comment:
Technically, it spawns a process to to find both the monaddrs and the cephx keys. This makes it sound a bit like it only does that for the keys.

Reply:
Umm... I just rearranged this paragraph in the doc. We should include the additional detail too.